addons/crm/views/crm_team_views.xml

```diff
@@ -224,6 +223,23 @@
             <field name="search_view_id" ref="crm.view_crm_case_opportunities_filter"/>
         </record>
 
+        <record id="action_lead_form" model="ir.actions.act_window">
```

Comment:
Guidelines: this should not be named like that, could you update it according to guidelines ?

Also most actions are defined on top of file (well, a bit the mess but at least we could try to keep it a bit organized). Could you move it below crm_lead_action_team_overdue_opportunity ?